.storybook/polaris-readme-loader.js

```diff
@@ -138,8 +138,8 @@ import {
   Heading,
   Icon,
   Image,
-  IndexProvider,
-  IndexTable,
+  UnstableIndexProvider as IndexProvider,
```

Comment:
Commit adding unstable prefix

Reply:
The instability is regarding the API since we've never revisited it. I was hoping to take a look at it this week but didn't have the chance. I'm going on vacation but ill take a look when I'm back and see if we can resolve the API